src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -150,4 +150,7 @@
   <data name="Gathering_Suggestions_Waiting_for_the_solution_to_fully_load" xml:space="preserve">
     <value>Gathering Suggestions - Waiting for the solution to fully load</value>
   </data>
+  <data name="LinesIndicator" xml:space="preserve">
```

Comment:
We usually name this something like Lines_0_to_1 so it's clearer in the use site.